src/tests/mocha/declaration-tests.ts

```diff
@@ -9,6 +9,7 @@ import { Event, Wildcard } from "../../lib/events";
 import { Fragment } from "../../lib/fragments";
 import { getResolvedDeclaration } from '../../lib/save-declarations';
 import { ParserOptions } from "../../lib/source-spec";
+import { Console } from "console";
```

Comment:
What's this import from?

Reply:
Result of testing. I'll need to delete it.

Reply:
that's from testing. I'll remove it.